src/views/HomePage/index.vue

```diff
@@ -50,12 +212,58 @@ export default defineComponent({
     .heya-search-input {
       width: 90%;
     }
+
+    .sidebar-toggle-button {
+      border: none;
+      border-radius: 10px;
+      background-color: #977b85;
+      color: #ffffff;
+      font-size: 12px;
+      padding: 2px 20px;
+      margin: 10px;
+      cursor: pointer;
+    }
+
+    .sidebar-text-button {
+      width: 180px;
+      padding: 0 10px;
+      margin: 0 auto;
+      border-bottom: thin solid #ffffff;
+      text-align: left;
+      font-size: 12px;
+      line-height: 3rem;
+      color: #ffffff;
+      cursor: pointer;
+    }
   }
 
-  .main {
-    overflow: auto;
+  .home-main {
     grid-column: 2;
     grid-row: 1;
+    overflow: auto;
+    text-align: center;
+    padding: 10px 0;
+
+    .heya-cards {
+      display: flex;
+      flex-wrap: wrap;
```

Comment:
```suggestion
      display: grid;
      grid-template-columns: repeat(auto-fill, 410px);
      justify-content: center;
```

こんな感じで (多分) うまくいきました

Reply:
あーできました
ありがとう 🙇